src/components/ResetPasswordForm/ResetPasswordForm.tsx

```diff
@@ -14,8 +14,10 @@ type Props = {
 const ResetPasswordForm: React.FC<Props> = ({ onCancel, onReset, submitting }: Props) => {
   const { t } = useTranslation('account');
   return (
-    <div className={styles.resetPassword}>
-      <h5 className={styles.title}>{t('reset.reset_password')}</h5>
+    <div className={styles.resetPassword} role="dialog" aria-labelledby="resetPassword">
```

Comment:
I would have chosen an id of `modalTitle` or something similar. But if multiple occurrences of this (modal) HTML can exist in the DOM, than I would definitely make it unique. No need for a change, just want to make you aware of this.

Reply:
Thanks, that is good feedback! On this page, in some configs, multiple dialogs can occur in the DOM. So on this page it's good practice